packages/clay-popover/src/index.tsx

```diff
@@ -69,6 +91,11 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 	 */
 	show?: boolean;
 
+	/**
+	 * Callback for setting the offset of the popover from the trigger.
+	 */
+	offset?: (points: Point) => [number, number];
```

Comment:
I will never infer offset will be a callback. ~~Maybe we should apply something like `onOffSet`~~

Reply:
That's true, I'm not sure about `onOffset`, maybe yes if I can't find another name I'll go with it.